packager/util/util_test.go

```diff
@@ -22,6 +23,33 @@ func TestCertName(t *testing.T) {
 	assert.Equal(t, "Qk83Jo8qB8cEtxfb_7eit0SWVt0pdj5e7oDCqEgf77o", util.CertName(pkgt.B3Certs[0]))
 }
 
+func TestGetDurationToExpiry(t *testing.T) {
+	// Time before the cert validity.
+	beforeCert := time.Date(2019, time.May, 8, 0, 0, 0, 0, time.UTC)
+	// Time after the cert validity.
+	afterCert := time.Date(2019, time.August, 8, 0, 0, 0, 0, time.UTC)
+	// Time 2 days before cert validity expiration.
+	twoDaysBeforeExpiry := time.Date(2019, time.August, 5, 5, 43, 32, 0, time.UTC)
+	// Time 0 days, 1 hour before cert validity expiration.
+	oneHourBeforeExpiry := time.Date(2019, time.August, 7, 4, 43, 32, 0, time.UTC)
+	// Time 0 days before cert validity expiration.
+	zeroDaysBeforeExpiry := time.Date(2019, time.August, 7, 5, 43, 32, 0, time.UTC)
+
+	d, err := util.GetDurationToExpiry(pkgt.B3Certs[0], beforeCert)
+	assert.Contains(t, errorFrom(err), "Certificate has invalid duration")
```

Comment:
Maybe use https://godoc.org/github.com/stretchr/testify/assert#EqualError ? Unless an exact string match is prohibitive.